Controller/Store/GetStoredetails.php

```diff
@@ -0,0 +1,195 @@
+        \Magento\Store\Model\StoreManagerInterface $storeManager, 
+\Magento\Framework\App\Action\Context $context,
+\Magento\Directory\Api\CountryInformationAcquirerInterface $countryInformation,
+\Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+
+    )
+    {        
+      
+        $this->_storeManager = $storeManager;   
+    
+	$this->countryInformation = $countryInformation;
+	$this->scopeConfig = $scopeConfig;
+
+        parent::__construct($context);
+    }
+
+    /**
+     * Retrieve formatted store address from config
+     *
+     * @param Store $store
+     * @return string
+     */
+    public function getFormattedAddress(Store $store)
+    {
+        /** @var \Magento\Customer\Block\Address\Renderer\DefaultRenderer $renderer */
+        $renderer = $this->addressConfig->getFormatByCode('html')->getRenderer();
+        return $renderer->renderArray($this->getStoreInformationObject($store)->getData());
+    }
+    
+    /**
+     * Get store identifier
+     *
+     * @return  int
+     */
+    public function getStoreId()
+    {
+        return $this->_storeManager->getStore()->getId();
+    }
+    
+    /**
+     * Get website identifier
+     *
+     * @return string|int|null
+     */
+    public function getWebsiteId()
+    {
+        return $this->_storeManager->getStore()->getWebsiteId();
+    }
+    
+    /**
+     * Get Store code
+     *
+     * @return string
+     */
+    public function getStoreCode()
+    {
+        return $this->_storeManager->getStore()->getCode();
+    }
+    
+    /**
+     * Get Store name
+     *
+     * @return string
+     */
+    public function getStoreName()
+    {
+        return $this->_storeManager->getStore()->getName();
+    }
+    
+    /**
+     * Get current url for store
+     *
+     * @param bool|string $fromStore Include/Exclude from_store parameter from URL
+     * @return string     
+     */
+    public function getStoreUrl($fromStore = true)
+    {
+        return $this->_storeManager->getStore()->getCurrentUrl($fromStore);
+    }
+    
+    /**
+     * Check if store is active
+     *
+     * @return boolean
+     */
+    public function isStoreActive()
+    {
+        return $this->_storeManager->getStore()->isActive();
+    }
+
+   public function execute()
+    {
+         $resultJson = $this->resultFactory->create(ResultFactory::TYPE_JSON);     
+        try{
+        	$storeid = $this->getStoreId();
+		$websiteid = $this->getWebsiteId();
+		$storecode = $this->getStoreCode();
+		$storename = $this->getStoreName();
+		$storeurl = $this->getStoreUrl();
+
+//$conf = $this->_helper->getStoreConfig('app/code/RicardoMartins/PagSeguro/etc/frontend/di.xml');
+
+$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+
+$directory = $objectManager->get('\Magento\Framework\Filesystem\DirectoryList');
+
+$rootPath  =  $directory->getRoot();
+
+$compojson = file_get_contents($rootPath.'/app/code/RicardoMartins/PagSeguro/composer.json');
```

Comment:
file_get_contents is not allowed this way in M2... 
besides that, the module will not exist under this folder except when in development. You may use ComponentRegistrar to get module path
Anyway, please, find a better way to get module's version please